src/main/java/ladder/domain/PlayResult.java

```diff
@@ -0,0 +1,40 @@
+package ladder.domain;
+
+import java.util.Objects;
+
+public class PlayResult {
+    private final String playResult;
+
+    public PlayResult(final String playResult) {
+        validateNullOrEmpty(playResult);
+        this.playResult = playResult;
+    }
+
+    private void validateNullOrEmpty(final String playResult) {
+        if (Objects.isNull(playResult) || playResult.isEmpty()) {
+            throw new IllegalArgumentException("실행 결과는 null 또는 empty 일 수 없습니다.");
+        }
+    }
+
+    public String getValue() {
+        return playResult;
+    }
+
+    @Override
+    public String toString() {
+        return String.valueOf(playResult);
```

Comment:
```suggestion
        return playResult;
```

String.valueOf로 처리하신 이유가 있을까요!?

Reply:
아닙니다. 실수로 저렇게 작성했네요ㅠ... 수정하겠습니다.